### PR TITLE
feat: add test for sensor service

### LINKED ISSUE #215: Implement test for sensor service

### PR DESCRIPTION
**Pull request by doriengr**
Ref #130 
Close #215

### CODE REVIEW

File: internal/service/domain/sensor/sensor_test.go [hunk 1]
```diff
@@ -0,0 +1,66 @@
+package sensor
+
+import (
+	"context"
+	"testing"
+
+	sensorUtils "github.com/green-ecolution/green-ecolution-backend/internal/server/http/handler/v1/sensor"
```
Comment (choffmann) [inline internal/service/domain/sensor/sensor_test.go:7]: Maybe this should be a separate package `sensorUtils` or `sensortest` (like `httptest` in standard lib)?

Comment (doriengr) [reply internal/service/domain/sensor/sensor_test.go:7]: We have those files also in the other folders. Maybe we can store the test data all in one file? I don't know what is the possible best way to handle those.

Comment (doriengr) [reply internal/service/domain/sensor/sensor_test.go:7]: 👍

File: internal/storage/postgres/sensor/get_test.go [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+		// when
+		got, err := r.GetSensorDataByID(ctx, 1)
+
+		// then
+		assert.Error(t, err)
+		assert.Nil(t, got)
+	})
+}
+
+func getTestSensors() []*entities.Sensor {
```
Comment (Zoka-tech) [inline internal/storage/postgres/sensor/get_test.go:325]: Wouldnt it be better to have the "fake" sensor data in a seperate file like with "internal/service/domain/tree/utils.go" for example?

**Comment by doriengr** [comment]
I am closing this PR. Due to the wrong rebase of mine it is not reviewable anymore. The sensor service tests is also implemented in #209

**Pull request closed without merging**